tests/bootstrap.php

```diff
@@ -32,6 +32,15 @@
 	$_test_root = '/tmp/wordpress-tests-lib';
 }
 
+// Check if tests are run locally without configuring test environment.
+if ( false !== strpos( $_test_root, 'wp-phpunit/wp-phpunit' ) ) {
+	if ( ! getenv( 'WP_PHPUNIT__TESTS_CONFIG' ) || ! getenv( 'WP_TESTS_CONFIG_FILE_PATH' ) ) {
```

Comment:
`WP_TESTS_CONFIG_FILE_PATH` is a constant.

Reply:
Thanks, I've updated it in the latest commit.